pkg/skaffold/build/builder_mux.go

```diff
@@ -128,6 +128,11 @@ func (b *BuilderMux) Build(ctx context.Context, out io.Writer, tags tag.ImageTag
 		return built, nil
 	}
 
+	err := checkMultiplatformHaveRegistry(b, artifacts, resolver)
+	if err != nil {
+		return nil, fmt.Errorf("%w", err)
```

Comment:
can you create an issue for why we're doing the single unwrapping in the `ShowAIError` function. And add a todo comment here:
```
	return nil, fmt.Errorf("%w", err) // TODO: remove error wrapping after fixing #9999
```